entrypoint.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn "todo_app.app:create_app()" --bind "0.0.0.0:${PORT}" "$@"
```

Comment:
Not vital here but a good practice to always get into adding [a shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) to the top of these files! E.g. `#!/bin/bash`